apps/daffio/src/app/docs/shared/services/docs-service.interface.ts

```diff
@@ -1,6 +1,9 @@
 import { DaffioDoc } from '../models/doc';
 import { Observable } from 'rxjs';
+import { DaffioDocList } from '../models/doc-list';
+import { DaffioGuideList } from '../models/guide-list';
 
-export interface DaffioDocServiceInterface<T extends DaffioDoc> {
+export interface DaffioDocServiceInterface<T extends DaffioDoc, V extends DaffioGuideList> {
   get(path: string): Observable<T>;
+  getGuideList(): Observable<V>
```

Comment:
Can you just repurpose `get` to do the same thing?

Reply:
It would not be trivial, because the `get` function expects a `DaffioDoc` and a `getGuideList`  expects a `DaffioGuideList`. This comes up in the http request.